.vscode/settings.json

```diff
@@ -6,4 +6,6 @@
         "swizzin",
         "xenial"
     ]
-}
\ No newline at end of file
+    "shellformat.flag": "-i 4",
+    "shellformat.path": ""
```

Comment:
This should be left out as that's relative for everyone. Leaving this unset will use plugin default, which either downloads the binary or uses it if it is in the path

Reply:
we are just talking about line 10? ` "shellformat.path": ""`